packages/cache/src/internal/blob/upload-cache.ts

```diff
@@ -0,0 +1,27 @@
+import * as core from '@actions/core'
+import {
+  BlobClient,
+  BlockBlobClient,
+  BlockBlobParallelUploadOptions
+} from '@azure/storage-blob'
+
+export async function UploadCacheFile(
+  signedUploadURL: string,
+  archivePath: string
+): Promise<{}> {
+  // Specify data transfer options
+  const uploadOptions: BlockBlobParallelUploadOptions = {
+    blockSize: 4 * 1024 * 1024, // 4 MiB max block size
+    concurrency: 4, // maximum number of parallel transfer workers
+    maxSingleShotSize: 8 * 1024 * 1024 // 8 MiB initial transfer size
+  }
```

Comment:
were these options derived from defaults?

Reply:
Yep, we need to tune them as we go